lib/logstash/outputs/rabbitmq.rb

```diff
@@ -76,20 +71,108 @@ def initialize(params)
     super
   end
 
-  # Use MarchHare on JRuby to avoid IO#select CPU spikes
-  # (see github.com/ruby-amqp/bunny/issues/95).
-  #
-  # On MRI, use Bunny.
-  #
-  # See http://rubybunny.info and http://rubymarchhare.info
-  # for the docs.
-  if RUBY_ENGINE == "jruby"
-    require "logstash/outputs/rabbitmq/march_hare"
+  public
```

Comment:
Just noticed @jordansissel's comment. OK, if this is intentional and used/enforced across all sub-projects, so be it.